src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -45,6 +45,14 @@ export function handleCLIException( exception: Error ) {
 		message = message.replace( 'ERROR: ', '' );
 
 		console.log( errorPrefix, message );
+
+		if ( ! process.env.DEBUG ) {
```

Comment:
Why not just output the full exception instead of hiding it behind `debug`? There's always a likelihood that when you re-run the command that the same issue is not reproducible.

Reply:
I considered it but I found Suso's counterpoint valid - namely, that as a public-facing tool it may be a bit much to display unconditionally, usually, that kind of a thing is hidden under debug/verbose flags, so I thought explicit instructions plus hiding behind `DEBUG` is a good compromise.